packages/typescript-plugin-vue/src/servers/routing.ts

```diff
@@ -260,6 +260,31 @@ function createLanguageServiceRouter(
           })
           .filter(isNotNull)
       },
+      getCompletionsAtPosition(fileName, position, options) {
+        return choose(fileName).getCompletionsAtPosition(
```

Comment:
We may need to remove references to virtual files from completions.

Reply:
how can I filter those?

Example of entries from the test:

```json
[
  {
    "name": "compile",
    "kind": "function",
    "kindModifiers": "export,declare",
    "sortText": "0"
  },
  {
    "name": "useCssModule",
    "kind": "function",
    "kindModifiers": "export,declare",
    "sortText": "0"
  },
  {
    "name": "useCssVars",
    "kind": "function",
    "kindModifiers": "export,declare",
    "sortText": "0"
  },
  {
    "name": "createApp",
    "kind": "const",
    "kindModifiers": "export,declare",
    "sortText": "0"
  },
  {
    "name": "createSSRApp",
    "kind": "const",
    "kindModifiers": "export,declare",
    "sortText": "0"
  },
  {
    "name": "hydrate",
    "kind": "const",
    "kindModifiers": "export,declare",
    "sortText": "0"
  },
  {
    "name": "render",
    "kind": "const",
    "kindModifiers": "export,declare",
    "sortText": "0"
  },
  {
    "name": "Transition",
    "kind": "const",
    "kindModifiers": "export,declare",
    "sortText": "0"
  },
  {
    "name": "TransitionGroup",
    "kind": "const",
    "kindModifiers": "export,declare",
    "sortText": "0"
  },
  {
    "name": "TransitionGroupProps",
    "kind": "type",
    "kindModifiers": "export,declare",
    "sortText": "0"
  }
]
```

Reply:
I don't think we need to filter these, we have to correct fileName for import suggestions from .vue (if any)

Reply:
added that 😎 
https://github.com/znck/vue-developer-experience/pull/31/commits/606cd0c4c586691d50aedad581a3ecf0fbe8c437